tests/unit/components/UI/Player/VideoPlayer.spec.js

```diff
@@ -38,5 +38,12 @@ describe("VideoPlayer.vue", () => {
     expect(initiatePlayer).toHaveBeenCalled();
     expect(removePlyrPoster).toHaveBeenCalled();
     expect(setPlayerProperties).toHaveBeenCalled();
+
+    await wrapper.setProps({
+      videoId: "qJWALEoGge4",
+    });
+    expect(initiatePlayer).toHaveBeenCalled();
+    expect(removePlyrPoster).toHaveBeenCalled();
+    expect(setPlayerProperties).toHaveBeenCalled();
```

Comment:
I think this should be part of a new test case that says - changing video ID updates the video in the player.